imports/ui/Task.html

```diff
@@ -2,6 +2,9 @@
   <li>
     <input type="checkbox" class="toggle-checked" checked="{{isChecked}}" />
     <span>{{text}}</span>
+    {{#if isUpdateLoading}}
+    <div class="loading">loading...</div>
```

Comment:
@AbegaM indentation

Reply:
@AbegaM add styling if necessary as well

Reply:
I have fixed the indentation and have added a simple animation for the spinner